types/replicatedservice/v2/replicatedservice.py

```diff
@@ -0,0 +1,194 @@
+"""Defines a ReplicatedService type by creating both a Service and an RC.
```

Comment:
It's hard for me to tell here what's changed that's causing this version bump. Should we have some kind of notes in type versions to understand these kinds of changes?